src/Opdex.Platform.Infrastructure.Abstractions/Data/Queries/Tokens/TokensCursor.cs

```diff
@@ -154,10 +154,15 @@ public enum TokenProvisionalFilter
     /// </summary>
     Provisional = 1,
 
+    /// <summary>
+    /// Opdex mined, staking tokens
+    /// </summary>
+    Staking = 2,
+
     /// <summary>
     /// All tokens.
     /// </summary>
-    All = 2
+    All = 3
```

Comment:
Maybe this should include `NonProvisional` and `Security` as well, for flexibility?

Reply:
Good catch, didn't realize NonProvisional wasn't an option there. Will go back through this. Anytime anything is marked as a security we will likely not return anything about it.

I'll keep it out for now but move `All` to 1 so we can append more options cleaner down the road if we decide.